tests/darwinScrublist.nix

```
@@ -45,6 +45,7 @@ let
     "gh-dash"
     "ghostty"
     "git"
+    "git-patdiff"
```

```suggestion
    "patdiff"
```

This is the name of the package from nixpkgs

---

Ah ha I see, thanks :)